Fix clang printf format security warning to make test compile.

Index: conformance/interfaces/pthread_mutex_trylock/testfrmw.c
--- conformance/interfaces/pthread_mutex_trylock/testfrmw.c.orig
+++ conformance/interfaces/pthread_mutex_trylock/testfrmw.c
@@ -54,7 +54,7 @@ void output( char * string, ... )
    nw = time(NULL);
    now = localtime(&nw);
    if (now == NULL)
-      printf(ts);
+      puts(ts);
    else
       printf("[%2.2d:%2.2d:%2.2d]", now->tm_hour, now->tm_min, now->tm_sec);
    va_start( ap, string);
